lib/analysis.js

```diff
@@ -52,6 +53,7 @@ function AnalysisFactory(DatabaseServiceClass) {
 
 AnalysisFactory.prototype.create = function(configuration, definition, callback) {
     debug('Using configuration=%j', configuration);
+    var logger = createLogger(configuration.logger).child({ analysis: definition.type, user: configuration.user });
```

Comment:
What approach fits better:

- Only one log per analysis graph
- One log per node inside the graph. 

I'd go with one log per graph. What do you think???